java-extensions/jdbc-bridge/src/main/java/com/starrocks/jdbcbridge/JDBCScanner.java

```diff
@@ -61,7 +60,7 @@ public JDBCScanner(String driverLocation, JDBCScanContext scanContext) {
 
     public void open() throws Exception {
         String cacheKey = computeCacheKey(scanContext.getUser(), scanContext.getPassword(), scanContext.getJdbcURL());
-        URL driverURL = new File(driverLocation).toURI().toURL();
+        URL driverURL = new URL(driverLocation);
```

Comment:
can u give out an example?

Reply:
oops, I mistake for the driverLocation is the 'driver_url' property of catalog which canbe 'http://...', actually it always be local file protocol on BE.